servers/intro_course/main.go

```
@@ -39,20 +39,15 @@
 	}
 
 	realm := utils.GetEnv("KEYCLOAK_REALM_NAME", "prompt")
-	clientID := utils.GetEnv("KEYCLOAK_CLIENT_ID", "prompt-server")
-	expectedAuthorizedParty := utils.GetEnv("KEYCLOAK_AUTHORIZED_PARTY", "prompt-client")
-
-	log.Info("Debugging: baseURL: ", baseURL, " realm: ", realm, " clientID: ", clientID, " expectedAuthorizedParty: ", expectedAuthorizedParty)
-
-	// first we initialize the keycloak token verifier
-	keycloakTokenVerifier.InitKeycloakTokenVerifier(context.Background(), baseURL, realm, clientID, expectedAuthorizedParty, queries)
+	coreURL := utils.GetCoreUrl()
+	promptSDK.InitAuthenticationMiddleware(baseURL, realm, coreURL)
```

golangci-lint (1.64.8)

Error return value of `promptSDK.InitAuthenticationMiddleware` is not checked

(errcheck)